documentation/src/main/markdown/integrations/jenkinsplugin/downloadingandinstalling.md

```diff
@@ -0,0 +1,34 @@
+# Downloading, Installing, and Updating the Plugin
+To install the [solution_name] for Jenkins plugin, perform the following steps:
+
+1. Navigate to **Manage Jenkins** > **Manage Plugins**.
+1. Select the **Available** tab.  Note that if the plugin is already installed, it does not appear in the **Available** list.
+1. Select **Synopsys Detect**.
+1. Click **Download now and install after restart**. This is the Synopsys recommendation for installing the plugin.
+1. After restarting Jenkins, confirm that the plugin is successfully installed by navigating to **Manage Jenkins** > **Manage Plugins > Installed**, and verify that **[solution_name]** displays in the list.
+
+[solution_name] plugin for Jenkins versions 3.0.0 and 3.1.0 releases are also available in the public [Artifactory](https://sig-repo.synopsys.com/webapp/#/artifacts/browse/tree/General/bds-integrations-release/com/blackducksoftware/integration/blackduck-detect/).
+
```

Comment:
Something to keep in the back of your mind for the future: We used to have artifactory URLs sprinkled throughout the Detect doc, and an IT dept that loved moving/renaming artifactory servers. We eventually centralized them on https://community.synopsys.com/s/document-item?bundleId=integrations-detect&topicId=downloadingandrunning/downloadlocations.html&_LANG=enus, put the artifactory details in a single source file (https://github.com/blackducksoftware/synopsys-detect/blob/master/buildSrc/src/main/java/com/synopsys/integration/detect/docs/content/Terms.java), and wrote some code to generate that doc page from a template. The IT group now seems to be busy with something other than renaming artifactory servers, so this may no longer be a significant issue. (In retrospect, generating that doc page from a template may have been overkill, but I'd say there was value, at least for a while, in having download URLs in a central location.)

Reply:
Hi Steve, I'd like to actually do this now for this new link, lest I lose track of this plan. ; )
I'll make the associated changes. Thanks!